.github/workflows/swift.yml

```diff
@@ -1,4 +1,4 @@
-name: Swift
+name: build
```

Comment:
Nit: maybe something like "test" or "RunTests" or so would be more descriptive? Also feel free to rename the file as well if that makes sense